index.html

```diff
@@ -33,26 +37,27 @@ <h2 class="header-light regular-pad">What is TeaVM?</h2>
         <div class="col-sm-4">
             <h1 class="text-center"><i class="fa fa-check-circle-o" aria-hidden="true"></i></h1>
             <h3 class="text-center">Easy to use</h3>
-            <p>Create a new project from Maven archetype and develop with pleasure.
-                You don't need complicated setup with npm, Webpack, UglifyJS, Babel, etc.
+            <p>Create a new project from a Maven archetype and develop with pleasure.
+                You don't need a complicated setup with npm, Webpack, UglifyJS, Babel, etc.
                 TeaVM does everything for you.
             </p>
         </div>
         <div class="col-sm-4">
             <h1 class="text-center"><i class="fa fa-cogs" aria-hidden="true"></i></h1>
             <h3 class="text-center">A powerful web framework included</h3>
             <p>With TeaVM, you can use Flavour, a framework for creating single-page web applications
-                in Java, Kotlin or Scala. Experience is quite close to Angular, but the framework
+                in Java, Kotlin or Scala. Templates, components, routing, REST API bindings, and more
```

Comment:
> Templates, components, routing, REST API bindings

I'm not agree with this.

1. What if number of features grows? I'll have to update this text as well.
2. Things like "components" can't be properly understood without context.

Reply:
Makes sense to me.  I will revert this line.

Reply:
OK, I made the change.  Let me know if there is anything else you'd like me to fix.